remove superfluous license in readme

ElsLommelen opened the pull request:
In the readme on the github page, there were 2 badges with mention of the licence.  I suggest in this PR to remove one (the one that did not link to a website other than one that just showed the badge itself, which is also the one that is not shown on the pkgdown website).

ThierryO commented:
Good catch. However I think we should remove the other badge and keep the one you removed. You've kept the one linked to a specific license (GPL 3.0) and removed the more generic one.

ElsLommelen commented:
Eh, I think I kept the GPL-3 and removed the GPL-3.0.  Moreover, the one that I removed, did not appear in the pkgdown website, so when removing the other one, the license will probably disappear in the pkgdown website.

ThierryO commented:
Closing this PR as it is handled in a2e3755 in different way.
